packages/instant/src/index.umd.ts

```diff
@@ -110,3 +110,7 @@ export const render = (config: ZeroExInstantConfig, selector: string = DEFAULT_Z
     };
     window.onpopstate = onPopStateHandler;
 };
+
+// Write version info to the exported object for debugging
+export const GitSha = process.env.GIT_SHA;
```

Comment:
nit but I would expect these to be in all caps constant-case `GIT_SHA`, but there may be an accepted pattern for this that 0x uses elsewhere?